src/modules/WizardsWardrobeBanking.lua

```diff
@@ -116,17 +116,13 @@ function WWB.DepositSetup(zone, pageId, index)
 	WWB.MoveItems(gearTable, bankBag)
 end
 
-function WWB.DepositAllSetups(skipCurrentCharacter)
-	local bankBag = GetBankingBag()
-	if WW.DISABLEDBAGS[bankBag] then return end
-	
-	local itemLocationTable = WW.GetItemLocation()
+function WWB.GetAccountSavedGear(skipCurrentCharacter)
+  local itemLocationTable = WW.GetItemLocation()

```

Comment:
Both your function and this function are not named very well. It sounds like they get item locations, when actually they get a list of all gear in the current character's inventory (and on the body) along with their bag slot.